CHANGES.rst

```diff
@@ -3,7 +3,86 @@
 0.12 (unreleased)
 -----------------
 
-- Planned for May 15, 2019
+Summary
++++++++
+
+- Released on May 30, 2019
+- 9 contributors
+- 2 months of work
+- 63 pull requests (not all listed below)
+
+**What's new**
+
+For Gammapy v0.12 we did our homework, cleaned-up the basement and emptied the
+trash bin. It is a smaller release that does not introduce many new features,
```

Comment:
"smaller" -> "small", not clear what you compare it to, it's a pretty big release for a 2 month dev cycle.

Reply:
Done